site/src/doc/quickstart.md

```diff
@@ -58,21 +58,22 @@ Open the `main.js` file in a text editor.  It should look something like this:
 
 ```js
 import './style.css';
-import {Map, View} from 'ol';
-import TileLayer from 'ol/layer/Tile';
-import OSM from 'ol/source/OSM';
+import Map from 'ol/Map.js';
```

Comment:
I was wondering if this change here was related to the problem this PR addresses.
```patch
-import {Map, View} from 'ol';
+import Map from 'ol/Map.js';
+import View from 'ol/View.js';
```